### PR TITLE
[10.0] Create issues and tasks from portal/website

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

**Comment by mileo** [comment]
cc: @ericflaux this feature can be copied to helpdesk. We can do it in a few days, in version 12 it is even simpler.

Best regards

**Comment by mileo** [comment]
@ananiasfilho please see travis issues:

https://travis-ci.org/OCA/project/jobs/601912955#L338-L352
and 
https://travis-ci.org/OCA/project/jobs/601912955#L357-L360

File: website_task/README.rst [hunk 1]
```diff
@@ -31,15 +31,9 @@ Usage
 
 To use this module, you need to:
 
-#. Go to ...
-
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```
Comment (mileo) [inline website_task/README.rst]: You can't remove this data.

**Comment by ananiasfilho** [comment]
Need review

File: website_portal_issue/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Website Project Issue',
+    'summary': """
+        Create issues from website""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'KMEE,Odoo Community Association (OCA)',
+    'website': 'www.kmee.com.br',
```
Comment (nikul-serpentcs) [inline website_portal_issue/__manifest__.py]: URL should be `https://github.com/OCA/project`

File: website_task/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Website Task',
+    'summary': """
+        Create tasks from website""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'KMEE,Odoo Community Association (OCA)',
+    'website': 'https://www.kmee.com.br',
```
Comment (nikul-serpentcs) [inline website_task/__manifest__.py]: same as here

**Review by nikul-serpentcs — COMMENTED** [review]
Minor change

**Pull request closed without merging**